package.json

```
@@ -14,7 +14,7 @@
   },
   "license": "MIT",
   "devDependencies": {
-    "@tc39/ecma262-biblio": "^2.0.2288",
+    "@tc39/ecma262-biblio": "2.0.2322",
```

if this is meant to pin it, then let's use `=` so it's explicit
```suggestion
    "@tc39/ecma262-biblio": "=2.0.2322",
```

---

Unfortunately `--save-exact` does not include the `=`, and I think having the `=` be removed by a step in the readme is likely to be confusing.

---

yeah true, until https://github.com/npm/rfcs/pull/547 is fixed we'll have to omit the `=`